static/js/navigation.js

```diff
@@ -0,0 +1,54 @@
+var navDropdowns = document.querySelectorAll('.p-navigation__dropdown-link');
+var dropdownWindow = document.getElementsByClassName('dropdown-window')[0];
+
+navDropdowns.forEach(function(dropdown) {
+  dropdown.addEventListener('click', function(event) {
+    var clickedDropdown = this;
+
+    if (dropdownWindow.classList.contains('u-hide--fade')) {
+      dropdownWindow.classList.remove('u-hide--fade');
+    }
+
+    navDropdowns.forEach(function(dropdown) {
+      var dropdownContent = document.getElementById(dropdown.id + "-content");
+
+      if (dropdown === clickedDropdown) {
+        if (dropdown.classList.contains('is-selected')) {
+          dropdown.classList.remove('is-selected');
+          dropdownWindow.classList.add('u-hide--fade');
+          dropdownContent.classList.add('u-hide--fade');
+          dropdownContent.style = "visibility: hidden;"
```

Comment:
Changing styles directly with JavaScript can be quite expensive. Can this be handled with a utility class?